src/test/kotlin/uk/gov/justice/digital/hmpps/prisonregister/integration/GetPrisonNames.kt

```diff
@@ -78,6 +80,101 @@ class GetPrisonNames : IntegrationTest() {
     }
   }
 
+  @Test
+  fun `should return prison active name id only`() {
```

Comment:
what do you gain from having this test when you test for exactly the same thing as part of the tests below with
`null, WDI`
?